content/docs/people/reading-material/book-flatrate.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Book Flatrate"
+description: >
+  This is the summary of how our book flatrate works
+confidentiality: public
+---
+## We have a Book flatrate!
+
+This means if you have a book that you want to read - Buy it and get it reimbursed by Giant Swarm.
+Buy it via Spendesk, if you want to buy it in a local store to support your local dealer you can reimburse it via Spendesk. (Spendesk will be THE tool of choice here)!
+
+Yes -  any book (also audiobooks) that you are interested in!
+
+Like always sharing is caring so make sure you share a summary or your learnings in #chat-books and if its something a lot of people might be interested - why not talk about it in a lunch & learn session.
+
+
+### Important!
+Unfortunately German tax law decided to make things difficult. Therefore the following rules apply:
+- Do not spend more than 44 EUR each month.
+- Only buy books between the 1st and the 20st of each month. Book flatrate is closed from the 21st till the end of each month.
+- Use the [Book Log](https://docs.google.com/spreadsheets/d/1aLbQcybAtjLY90r7v62F0HT7D1nIYOnIWeO3IbI8vt8/edit#gid=24313467) to add your purchased books.
```

Comment:
We should probably remove all those extra parameters from links (like `/edit#gid=24313467`).